doc/specifications/proposed/content_download/content_download.md

```diff
@@ -41,10 +41,12 @@ Example: `/content/download/68/file/My-file.pdf`
 
 - language (optional)
 
+  > Should we keep this, given that the fileId binds to a particular language ?
```

Comment:
Yep. I first considered maintaining the TranslationHelper in the DownloadController:
- get the field identifier based on the field id and content
- use the translation helper to get the field in the right language
- send that

But the consequences were at best unclear.